invenio_notifications/backends/loaders.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
minor: we can rename this to `utils.py`

Reply:
It's now in `utils/loaders.py`. Since it's not a backend, leaving it in the parent folder felt wrong.